documentation/source/conf.py

```diff
@@ -0,0 +1,56 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+sys.path.insert(0, os.path.abspath('../../infertrade/'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'Infertrade'
+copyright = '2021, Furqan Hashim'
+author = 'Furqan Hashim'
+
+# The full version, including alpha/beta/rc tags
+release = '0.0.1'
```

Comment:
maybe 
```
import _version
release = _version.__version__
```
?

Reply:
@fernando-az-infertrade I've made suggested changes in commit [ef668a3](https://github.com/ta-oliver/infertrade/pull/103/commits/ef668a326713e89d3db2df709417dbcb48c654cd)

Reply:
@fhashim , thank you.